<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Add</class>
 <widget class="QDialog" name="Dialog_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_add">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Penghuni</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_id">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Kamar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_alamat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_idkmr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Penghuni</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alamat </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No HP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_noHp">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>130</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_nama">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No Kamar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_kmr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_biayaTambahan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>213</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Biaya Tambahan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_totBiaya">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_totBiaya">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Biaya</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
